scripts/jenkins-test

```diff
@@ -213,10 +213,11 @@ then
 	    export SPARKR_SUBMIT_ARGS="--jars ${ASSEMBLY_DIR}/${ASSEMBLY_JAR} --driver-class-path ${ASSEMBLY_DIR}/${ASSEMBLY_JAR} sparkr-shell"
 	    
             # we can run the python build, now that we have a spark executable
+	    # -DskipTests appears to skip the python/r tests?
 	    mvn -U \
 		-P python,r \
-		package \
-		-DskipTests \
+		test \
```

Comment:
I seem to recall skipping python tests was intentional?

Reply:
Why would we want to skip python tests? This would result in merging broken code.

Reply:
I dunno, git blame points to commit https://github.com/bigdatagenomics/adam/commit/7e92b9dcd475deecf6ed7c71761dbfcc6b16a9cf and before that commit https://github.com/bigdatagenomics/adam/commit/640c44bae1841217f981899c3a2b793c7ae06dcd and before that commit https://github.com/bigdatagenomics/adam/commit/c34a44004f80d4958df40eed6ae5d3760b8317ae, so it's always been that way

Reply:
Nope, not intentional, just a goof on my end.